src/components/search/Autocomplete.tsx

```diff
@@ -8,19 +8,22 @@ import classNames from 'classnames'
 interface AutocompleteProps extends Partial<AutocompleteOptions<any>> {
   isMobile: boolean
   containerClassname?: string
+  forceFocus: boolean
 }
 /**
  * Autocomplete widget based on Algolia Autocomplete
  * @param props
  * @returns
  */
-export const Autocomplete = (props: AutocompleteProps): JSX.Element => {
+export const Autocomplete = ({ forceFocus = false, ...otherProps }: AutocompleteProps): JSX.Element => {
   const containerRef = useRef<HTMLDivElement>(null)
   useEffect(() => {
-    setTimeout(() => {
-      const element = document.querySelectorAll(`.${props.id as string} .aa-Input`)[0] as HTMLInputElement
-      element?.focus()
-    }, 200)
+    if (forceFocus) {
```

Comment:
@osaf96 I make the main search box's autofocus optional (disabled by default).  Otherwise it will steal the focus from the slide viewer, preventing hot key from working properly.